frontend/src/container/LiveLogs/FiltersInput/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Col } from 'antd';
+import { initialQueriesMap } from 'constants/queryBuilder';
+import { useQueryBuilder } from 'hooks/queryBuilder/useQueryBuilder';
+import { useCallback, useMemo } from 'react';
+import {
+	IBuilderQuery,
+	TagFilter,
+} from 'types/api/queryBuilder/queryBuilderData';
+
+import {
+	ContainerStyled,
+	FilterSearchInputStyled,
+	SearchButtonStyled,
+} from './styles';
+
+function FiltersInput(): JSX.Element {
+	const {
+		stagedQuery,
+		handleSetQueryData,
+		redirectWithQueryBuilderData,
+		currentQuery,
+	} = useQueryBuilder();
+
+	const handleChange = useCallback(
+		(filters: TagFilter) => {
+			if (!stagedQuery) return;
+			const currentStagedQuery = stagedQuery.builder.queryData[0];
```

Comment:
Use a safe navigation operator and a default value

Reply:
@YounixM I think the code looks cleaner without a safe navigation operator and use guard logic on the top of function body